config/database.yml

```diff
@@ -20,8 +20,8 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  # username: postgres
-  # password: 2352122Aa.
+  username: postgres
+  password: momo123456
```

Comment:
- [ ] Good work implementing the features. ✔️ However, please kindly ensure to comment your custom username and password. 🤓

Reply:
thanks a lot already did that